addons/web/static/src/core/dropdown/dropdown_navigation_hook.js

```diff
@@ -86,7 +86,8 @@ export function useDropdownNavigation() {
         /** @type {NodeListOf<HTMLElement>} */
         const queryResult = menuRef.el.querySelectorAll(MENU_ELEMENTS_SELECTORS.join());
         for (const el of queryResult) {
-            const isSubDropdown = el.classList.contains("dropdown");
+            const isSubDropdown =
```

Comment:
can you explain this change?

Reply:
Oh sorry this change can be removed. Initially I used the SelectMenu component instead of <select> element, and there was an issue since the main dropdown would close when selecting an option from the component